com.netxms.NetXMSClient.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: com.netxms.NetXMSClient
+runtime: org.freedesktop.Sdk
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: netxms-client.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
```

Comment:
does this really support wayland? My experience with Java is that it usually doesn't.